src/app/teacher/authoring-tool.module.ts

```diff
@@ -35,6 +37,8 @@ import { EditNodeRubricComponent } from '../../assets/wise5/authoringTool/node/e
     AddYourOwnNode,
     AdvancedProjectAuthoringComponent,
     CardSelectorComponent,
+    ChooseImportComponentComponent,
+    ChooseImportComponentLocationComponent,
```

Comment:
Can you create an ImportComponentModule, put these components there, and import it in this module (AuthoringToolModule)?